suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -11,12 +16,74 @@ const overlayWrapperStyle = prepareNativeStyle(utils => ({
     backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation0,
 }));
 
+const bottomWrapperStyle = prepareNativeStyle(() => ({
+    position: 'absolute',
+    bottom: 40,
+    left: 0,
+    right: 0,
+    alignItems: 'center',
+}));
+
+const touchAreaStyle = prepareNativeStyle(utils => ({
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: utils.spacings.extraSmall,
+}));
+
+const getBiometricsTranslationKey = ({
+    isFacialEnabled,
+    isFingerprintEnabled,
+}: {
+    isFacialEnabled: boolean;
+    isFingerprintEnabled: boolean;
+}): TxKeyPath => {
+    if (Platform.OS === 'ios') {
+        return isFacialEnabled ? 'biometrics.ios.faceId' : 'biometrics.ios.touchId';
+    }
+
+    if (Platform.OS === 'android') {
+        if (isFingerprintEnabled && isFacialEnabled) return 'biometrics.android.combined';
+
+        if (isFingerprintEnabled) return 'biometrics.android.fingerprint';
+
+        if (isFacialEnabled) {
+            return 'biometrics.android.facial';
+        }
+    }
+
+    return 'biometrics.unknown';
+};
+
 export const BiometricOverlay = () => {
     const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
+    const { isFacialEnabled, isFingerprintEnabled } = useBiometricsSettings();
+    const { isBiometricsAuthenticationCanceled, setIsBiometricsAuthenticationCanceled } =
+        useIsBiometricsAuthenticationCanceled();
+
+    const handleReenable = () => {
+        // Setting this to true lets useBiometrics to ask for biometrics in case it was canceled by user before
+        // https://github.com/trezor/trezor-suite/issues/10647
+        setIsBiometricsAuthenticationCanceled(false);
+    };
+    const titleTransKey = getBiometricsTranslationKey({ isFacialEnabled, isFingerprintEnabled });
 
     return (
         <Box style={applyStyle(overlayWrapperStyle)}>
-            <Icon name="trezor" size="extraLarge" color="iconOnPrimary" />
+            <Icon name="trezor" size="extraLarge" color="iconDefault" />
+            {isBiometricsAuthenticationCanceled && (
+                <View style={applyStyle(bottomWrapperStyle)}>
+                    <TouchableOpacity onPress={handleReenable} style={applyStyle(touchAreaStyle)}>
+                        <BiometricsIcons
+                            isFacialEnabled={isFacialEnabled}
+                            isFingerprintEnabled={isFingerprintEnabled}
```

Comment:
These could be extracted from the hook from inside the component

Reply:
492763e1d2